### DISCUSSION TITLE
Persistent Advanced Options (between actions)

### DISCUSSION THREAD

**Discussion by yanayz** [mikekazakov/nimble-commander#356] [Ideas]
Would be great if the preferences in "advanced options" (e.g. when copying files) remain as the user sets them between actions.

Currently I have to reapply the same desired settings every single time I copy from my server, which isn't great...

![advanced options](https://github.com/user-attachments/assets/80b24ceb-3455-4f60-ae3f-e08f78ed29f4)

**Comment by mikekazakov** [comment]
Agree, it would be a welcome addition.
